### PR TITLE
Resolves #219 - missing file-destructive-push in python script

### LINKED ISSUE #219: Missing file-destructive-push in python script
The solenopsis python script is missing the file-destructive-push option. This works in bsolenopsis and is present as a target in the 1.1 version of solenopsis-build.xml.

### PR DESCRIPTION
**Pull request by lordarcy**
Add file-destructive-push argument
Add --pushfiles argument to specify list of files to push
Add --removefiles argument to specify list of files to remove

I will update the wiki at https://github.com/solenopsis/Solenopsis/wiki/1.1-Usage---Python once the code has been merged.

### CODE REVIEW

File: scripts/lib/ant.py [hunk 1]
```diff
@@ -104,27 +135,20 @@ def filePush(fileList):
 
     fileList - An array of file names to push
     """
-    if len(fileList) == 0:
-        logger.critical('No files listed to push')
-        sys.exit(-1)
+    processFileList(fileList, 'sf.files2push')
+    print getFlags()
```
Comment (pcon) [inline scripts/lib/ant.py]: I don't think we want to be printing this out

File: scripts/lib/ant.py [hunk 2]
```diff
@@ -104,27 +135,20 @@ def filePush(fileList):
...
-        runAnt('file-push')
-    else:
-        logger.critical('Unable to find any files to push.')
-        sys.exit(-1)
+    fileList - An array of file names to remove
+    
+    The list of files to push shold have already been set in the sf.files2push flag
+    """
+    processFileList(fileList, 'sf.files2remove')
+    print getFlags()
```
Comment (pcon) [inline scripts/lib/ant.py]: Same as line 139

**Comment by pcon** [comment]
If you can please remove those two prints and re-commit I think it'll be good

**Comment by lordarcy** [comment]
There was a print getFlags() in the original filePush(); I left it there when I refactored and added a matching one in the new fileDestructivePush(). I assumed they were there for debugging the flags that were passed to the ant build. Are you sure you want me to remove them? Those are the only two commands that have the flags printed out.

**Comment by pcon** [comment]
Yes, please remove it. It was an oversight that the print was left in the original file-push.

**Comment by pcon** [comment]
@lordarcy crud.  I totally missed this PR.  If you can rebase and fix the conflicts, I'll merge it.

**Comment by pcon** [comment]
Any chance of a rebase and conflict resolution?

**Comment by lordarcy** [comment]
@pcon I will work on this next week

**Comment by pcon** [comment]
This was done with 374dc5bfd09cee08aa1d0de08f739ed476de1032 to fix #248

**Pull request closed without merging**